Source/OCMock/OCPartialMockObject.m

```diff
@@ -80,6 +80,28 @@ - (Class)classToSubclassForObject:(id)object
     return object_getClass(object);
 }
 
+- (BOOL)hasSetupForwarderForSelector:(SEL)sel
+{
```

Comment:
If I'm not mistaken there is a much simpler way of figuring this out. The first few lines in `setupForwarderForSelector:` should do the trick, right?

Reply:
You are correct. Don't know why I went crazy here :)